HomeTask3.playground/Contents.swift

```diff
@@ -0,0 +1,67 @@
+import Darwin
+//Создать массив из кортежей (Int, String).
+
+let tuplesArray = [(1, "My name ist Valdemar"), (3, "Three"), (12, "Elizabeth"), (88, "Hello")]
+var tuplesArrayWithSquaredNumbers : [(Int, String)] = []
+
+//1. Пройти по элементам с помощью цикла for-in, создав Range от 0 до количества элементов массива, и возвести все Int в квадрат. (Лучше создать новый массив и результаты записать туда).
+
+for range in 0..<tuplesArray.count {
+    let squaring = powNumber(tuplesArray[range].0)
+    
+    tuplesArrayWithSquaredNumbers.insert((squaring,tuplesArray[range].1), at: tuplesArrayWithSquaredNumbers.endIndex)
```

Comment:
You can just use `tuplesArrayWithSquaredNumbers.append` instead of `insert` and end index

Reply:
Please, add a space after `,` that's between `squaring` and `tuplesArray[range].1`

Reply:
is was a deep night , and i do my homework )
My attention is decrease because of this 
I'll fix this